### PR TITLE
No longer make PySide and PyQt4 mandatory for PH5 installation

### PR DESCRIPTION
**Pull request by nick-iris**
### What does this PR do?
Adds to PR #326 . Removed mandatory requirement for PySide and PyQt4. These are now only required for running the GUI applications pforma, experiment_t_gen, and noven.

### Checklist
- [x] This PR is not directly related to an existing issue (which has no PR yet).
- [x] All tests pass.
- [x] Any new or changed features have are documented.
- [x] Changes have been added to `CHANGELOG.txt` .
- [x] First time contributors have added your name to `CONTRIBUTORS.txt` .

### CODE REVIEW

**Comment by dthomas1953** [comment]
Doing a test install now.

**Comment by dthomas1953** [comment]
Looking good.
```----------------------------------------------------------------------
Ran 23 tests in 14.309s

OK```
Compile went well. When I earlier had tried removing the try/except clauses for PyQt4 and PySide, I had re-included them in the install_requires section. Looks like they shouldn't have been there, either.

**Comment by nick-iris** [comment]
Okay I'll go ahead and merge

**Comment by nick-iris** [comment]
Thanks for the review

File: ph5/core/pmonitor.py [hunk 1]
```diff
@@ -26,14 +26,16 @@
...
 try:
-    from PySide import QtGui, QtCore
-except Exception as e:
-    LOGGER.error("No PySide: {0}".format(e.message))
-    try:
-        from PyQt4 import QtGui, QtCore
-        QtCore.Signal = QtCore.pyqtSignal
-    except Exception:
-        LOGGER.error("PyQt4 or PySide required: {0}".format(e.message))
+    from PyQt4 import QtGui, QtCore
```
Comment (dsentinel) [inline ph5/core/pmonitor.py]: I don't think this import should change.

Comment (nick-iris) [reply ph5/core/pmonitor.py]: Sure, I'll work on this now.

**Pull request merged**